bin/video2x_setup.py

```diff
@@ -87,6 +89,7 @@ def run(self):
     def _install_python_requirements(self):
         """ Read requirements.txt and return its content
         """
+        pip_uninstall('requirements.txt')
```

Comment:
why uninstall what might be already available?

Reply:
it was in there for my own purposes 
idk why i committed it in (removed now)